access/access.go

```diff
@@ -314,6 +321,26 @@ func (c Client) LatestBlockHeader(ctx context.Context) (*entities.BlockHeader, e
 	return resp.Block, nil
 }
 
+// LatestFinalizedBlockHeader returns the block header for the most recently
+// finalized block, i.e. one which hasn't been sealed yet.
+func (c Client) LatestFinalizedBlockHeader(ctx context.Context) (*entities.BlockHeader, error) {
+	ctx, span := c.newSpan(ctx, "flow.access_api.LatestFinalizedBlockHeader")
+	ctx, cancel := context.WithTimeout(ctx, defaultTimeout)
```

Comment:
do we need to have a config to control this default timeout?

Reply:
We currently use a `30s` timeout as the `defaultTimeout`. According to our traces from the last hour, the max any Access API call has taken in production is `4s`.

So, while we may want to make this configurable in the future, there doesn't seem any pressing need right now?